src/graphhopper/router.js

```diff
@@ -3,6 +3,7 @@ import graphHopperClient from './client.js';
 import logger from '../lib/logger.js';
 import * as gtfsRtClient from '../gtfs-rt/client.js';
 import { mergeAlertsIntoRoutes } from '../gtfs-rt/alerts.js';
+import { replacePtRoutlinesWithHighres } from '../route-linestring/client.js';
```

Comment:
This file path doesn't make sense

The convention in web-app is that directories correspond to a backend service (nominatim, graphhopper, photon, gtfs-rt, etc) and `src/someservice/client.js` is the HTTP client of that service

This is a utility function, how about under `lib`, like `src/lib/routelines.js`?

Also can you spell out `RouteLines` rather than `Routlines`?